src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -105,9 +105,16 @@ def __call__(
         if accepts_eta:
             extra_step_kwargs["eta"] = eta
 
-        for t in tqdm(self.scheduler.timesteps):
+        self.scheduler.set_timesteps(num_inference_steps)
+        if isinstance(self.scheduler, LmsDiscreteScheduler):
+            latents = latents * self.scheduler.sigmas[0]
+
+        for i, t in tqdm(enumerate(self.scheduler.timesteps)):
             # expand the latents if we are doing classifier free guidance
             latent_model_input = torch.cat([latents] * 2) if do_classifier_free_guidance else latents
+            if isinstance(self.scheduler, LmsDiscreteScheduler):
+                sigma = self.scheduler.sigmas[i]
+                latent_model_input = latent_model_input / ((sigma**2 + 1) ** 0.5)
```

Comment:
We'll probably need a mechanism to specify custom coefficients for inputs in the future, since the Karras scheduler needs it too.